openmetadata-ui/src/main/resources/ui/public/locales/en-US/Database/Oracle.md

```diff
@@ -12,14 +16,20 @@ CREATE USER user_name IDENTIFIED BY admin_password;
 -- CREATE ROLE
 CREATE ROLE new_role;
 
--- GRANT ROLE TO USER
+-- GRANT ROLE TO USER 
 GRANT new_role TO user_name;
 
 -- GRANT CREATE SESSION PRIVILEGE TO USER
 GRANT CREATE SESSION TO new_role;
+
 ```
 
-**Important:** To fetch metadata from Oracle database we use `python-oracledb`. This library only supports Oracle 12c, 18c, 19c and 21c versions!
+2. `GRANT SELECT` on the relevant tables which are to be ingested into OpenMetadata to the user
+```sql
+
+GRANT SELECT ON table_name TO {user | role};
+
+```
 
 You can find further information on the Oracle connector in the [docs](https://docs.open-metadata.org/connectors/database/oracle).
 
```

Comment:
@ayush-shah we need to update this for the documentation as well. We are also missing the information for the profiler and data quality and other workflow if it supports it 

example -> https://github.com/open-metadata/OpenMetadata/blob/main/openmetadata-ui/src/main/resources/ui/public/locales/en-US/Database/Redshift.md